pkg/bff/client.go

```diff
@@ -13,20 +14,36 @@ import (
 // GlobalDirectoryClient is a unified interface which can be implemented to create a
 // unified client to multiple GDS services.
 type GlobalDirectoryClient interface {
+	admin.DirectoryAdministrationClient
```

Comment:
I liked the idea of including the admin client in the unified interface as well, although it did cause some method collisions. However since it's HTTP, I can also see it being separated from the other two. What do you think?